src/Frontend/src/views/throughputreport/throughputClient.ts

```diff
@@ -31,8 +30,8 @@ class ThroughputClient {
     return data;
   }
 
-  public async reportAvailable() {
-    if (isThroughputSupported.value) {
+  public async reportAvailable(isThroughputSupported: boolean) {
+    if (isThroughputSupported) {
```

Comment:
I feel this makes more sense for the consumer of the api, not to do that call at all if `isThroughputSupported =false`, and hence we don't need this argument.